client/blocks/simple-site-rename-form/index.jsx

```diff
@@ -239,10 +239,11 @@ export class SimpleSiteRenameForm extends Component {
 							placeholder={ currentDomainPrefix }
 							isError={ shouldShowValidationMessage && ! isAvailable }
 						/>
-						{ shouldShowValidationMessage &&
-							validationMessage && (
-								<FormInputValidation isError={ ! isAvailable } text={ validationMessage } />
-							) }
+						<FormInputValidation
+							isHidden={ ! shouldShowValidationMessage }
+							isError={ ! isAvailable }
+							text={ validationMessage || '#' }
```

Comment:
'#' just for demo purposes, because there needs to something for the height not to collapse. If this is the route we'll take this can be improved on.